frog/imports/client/AccountModal/SignUp.js

```diff
@@ -0,0 +1,210 @@
+// @flow
+import React from 'react';
+import Avatar from '@material-ui/core/Avatar';
+import Button from '@material-ui/core/Button';
+import CssBaseline from '@material-ui/core/CssBaseline';
+import TextField from '@material-ui/core/TextField';
+import Link from '@material-ui/core/Link';
+import Grid from '@material-ui/core/Grid';
+import LockOutlinedIcon from '@material-ui/icons/LockOutlined';
+import Typography from '@material-ui/core/Typography';
+import Container from '@material-ui/core/Container';
+import { Meteor } from 'meteor/meteor';
+import { withStyles } from '@material-ui/styles';
+import {
+  errorBasedOnChars,
+  emailErrors,
+  passwordErrors
+} from './validationHelpers';
+
+import { type SignUpStateT, type SignUpPropsT } from './types';
+
+const styles = (theme: Object) => ({
+  '@global': {
+    body: {
+      backgroundColor: theme.palette.common.white
+    }
+  },
+  paper: {
+    marginTop: theme.spacing(8),
+    display: 'flex',
+    flexDirection: 'column',
+    alignItems: 'center'
+  },
+  avatar: {
+    margin: theme.spacing(1),
+    backgroundColor: theme.palette.secondary.main
+  },
+  form: {
+    width: '100%', // Fix IE 11 issue.
+    marginTop: theme.spacing(3)
+  },
+  submit: {
+    margin: theme.spacing(3, 0, 2)
```

Comment:
NIT: I think this is equivalent to `theme.spacing(3,0,2,0)`? Normally in CSS, we see 1, 2 or 4 values for margin, but 3 is non-standard.

Reply:
I have left it the way it is.. the spacing looks good to me.